### PR TITLE
bitrise-aws-device-farm-runner 0.0.8

### PR DESCRIPTION
**Pull request by lautarochiarle**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by adborbas** [comment]
Hi @lautarochiarle,

There is step with the same name at https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/aws-device-farm-runner with latest version of 0.0.7

Should this be an update to that step or you intended to create a new step?

**Comment by lautarochiarle** [comment]
hi @adborbas , 

thanks for your reply.

It is an update on that step ( https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/aws-device-farm-runner ), his owner is not maintaining it any more... we can add him to this thread so we are all on the same page. His name is @fadookie

**Comment by fadookie** [comment]
Yes, I asked lautarochiarle to make this new release himself after I merged his PR because I do not have time to maintain this step anymore and have not been able to find a new maintainer yet.

**Comment by trapacska** [comment]
Thanks! We are discussing this change with the team. Will take some more time. 🚀

**Comment by adborbas** [comment]
Hi @lautarochiarle,

We've come to a conclusion. Please reopen the PR as a new version to the original step https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/aws-device-farm-runner. 

Merging this PR would effectively create a new step which would lead to confusion as there is no need to create a new step, your intentions were only to change the maintainer of the step.

File: steps/bitrise-aws-device-farm-runner/0.0.8/step.yml [hunk 1]
```diff
@@ -0,0 +1,234 @@
+title: Amazon Device Farm Runner
+summary: Amazon Device Farm Runner
+description: Deploys app to device farm and starts a test run. [Please see the wiki
+  for setup instructions](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/wiki).
+website: https://github.com/lautarochiarle/bitrise-aws-device-farm-runner
+source_code_url: https://github.com/lautarochiarle/bitrise-aws-device-farm-runner
+support_url: https://github.com/lautarochiarle/bitrise-aws-device-farm-runner/issues
+published_at: 2020-07-15T15:51:27.398703558-03:00
+source:
+  git: https://github.com/lautarochiarle/bitrise-aws-device-farm-runner.git
```
[the remark this answers is not in the document]
Comment (fadookie) [reply steps/bitrise-aws-device-farm-runner/0.0.8/step.yml:10]: sorry i thought i had submitted this review earlier but it was in "pending" state.

**Comment by lautarochiarle** [comment]
Hi all,

I've created a new PR:
https://github.com/bitrise-io/bitrise-steplib/pull/2597

Hope it works this time.

Regards,

[image: Flux IT] <http://www.fluxit.com.ar/>
www.fluxit.com.ar
*Lautaro Chiarle *
*Solution Architect*
lautaro.chiarle@fluxit.com.ar  <lautaro.chiarle@fluxit.com.ar>

Camino Centenario #2565
La Plata / Buenos Aires / Argentina
(+54) 011 52726390

**Comment by adborbas** [comment]
Hi @lautarochiarle ,

We can pick up the topic at the new PR. You can go ahead and close this one.

**Pull request closed without merging**